src/back/hlsl/writer.rs

```diff
@@ -2190,6 +2190,7 @@ impl<'a, W: fmt::Write> super::Writer<'a, W> {
                     Asincosh { is_sin: bool },
                     Atanh,
                     Regular(&'static str),
+                    UintWrapped(&'static str),
```

Comment:
Could we name this `NoIntOverload` or `MissingIntOverload`? What do you think?

Reply:
I was trying to think of a good name, that sounds much better!